### ISSUE TITLE
Create Brazilian Portuguese translation on GitLocalize

### ISSUE THREAD

**Issue by ErickPetru** [vuejs/vue-ssr-docs#182]
Hi, I'm the translation leader of the Vue.js brazilian community. We're aiming to translate this SSR documentation and already started doing it as a fork, but it would make far mor sense to do it right here. Could you please create the `pt-br` folder in the `docs` and assign it to me in GitLocalize?

**Comment by kazupon** [comment]
@ErickPetru 
see the #227
gitlocalize is no longer available soon.
You can translate by GitHub's workflow.

**Comment by ErickPetru** [comment]
Hi @kazupon. So the right way will be following the instructions at https://vuepress.vuejs.org/guide/i18n.html, am I correct?

**Comment by kazupon** [comment]
@ErickPetru 
Yes! You can see vue-ssr-docs documentation structure at `docs` directory
